stubs/passlib/passlib/context.pyi

```diff
@@ -25,45 +25,53 @@ class CryptPolicy:
 
 class CryptContext:
     @classmethod
-    def from_string(cls, source, section: str = ..., encoding: str = ...): ...
+    def from_string(cls, source: str | bytes, section: str = ..., encoding: str = ...) -> CryptContext: ...
     @classmethod
-    def from_path(cls, path, section: str = ..., encoding: str = ...): ...
-    def copy(self, **kwds): ...
-    def using(self, **kwds): ...
+    def from_path(cls, path: str, section: str = ..., encoding: str = ...) -> CryptContext: ...
+    def copy(self, **kwds: Any) -> CryptContext: ...
+    def using(self, **kwds: Any) -> CryptContext: ...
     def replace(self, **kwds): ...
     def __init__(self, schemes: Any | None = ..., policy=..., _autoload: bool = ..., **kwds) -> None: ...
-    policy: Any
-    def load_path(self, path, update: bool = ..., section: str = ..., encoding: str = ...): ...
-    def load(self, source, update: bool = ..., section: str = ..., encoding: str = ...) -> None: ...
-    def update(self, *args, **kwds) -> None: ...
+    policy: CryptPolicy
+    def load_path(self, path: str, update: bool = ..., section: str = ..., encoding: str = ...) -> None: ...
+    def load(
+        self, source: str | bytes | dict[str, Any] | CryptContext, update: bool = ..., section: str = ..., encoding: str = ...
```

Comment:
Instead of `dict`, we can use `_typeshed.SupportsItems` here.

Reply:
Done https://github.com/python/typeshed/pull/8237/commits/d70e371641f963b9606d9928f70cd16ff97b01b7